lib/libzpool/uzfs_mgmt.c

```diff
@@ -283,6 +284,8 @@ uzfs_open_dataset_init(spa_t *spa, const char *ds_name, zvol_state_t **z)
 
 	zfs_rlock_init(&zv->zv_range_lock);
 	zfs_rlock_init(&zv->zv_mrange_lock);
+	mutex_init(&zv->io_tree_mtx, NULL, MUTEX_DEFAULT, NULL);
```

Comment:
These can be created/initialised  when replica receive START_REBUILD cmd from iSCSI controller.

Reply:
yes. Once we have that flow, we can shift this initialization there. As of now, that part is not yet implemented, so for working flow only i kept it here.